docs/sources/installation/docker.md

```diff
@@ -249,6 +282,10 @@ chown -R root:root /etc/grafana && \
   chown -R grafana:grafana /usr/share/grafana
 ```
 
+## Migration from a previous version of the docker container to 6.4 or later
+
+Grafana’s docker image was changed to be based on [Alpine](http://alpinelinux.org) instead of [Ubuntu](https://ubuntu.com/).
```

Comment:
Mgiht want to mention that we have both ubuntu and alpine from 6.5 and on

Reply:
Good point :+1: